### PR TITLE
pricing reference issue

### PR DESCRIPTION
**Pull request by reactionic127**
Fix for #254

### CODE REVIEW

File: src/templates/pricing.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+const Pricing = () => <Wrapper>Price</Wrapper>
+
+export default Pricing
+
+export const PricingQuery = graphql`
+  query price($path: String!) {
+    markdownRemark(frontmatter: { path: { eq: $path } }) {
+      frontmatter {
+        mission
```
Comment (jgwerner) [inline src/templates/pricing.js]: Isn't mission part of `contact.js`?

Comment (jgwerner) [reply src/templates/pricing.js]: We need to add `title`, `note`, `announcement`, don't we?

File: src/templates/pricing.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import styled from 'styled-components'
+import Link, { withPrefix } from 'gatsby-link'
+
+const Wrapper = styled.div`
+  padding-top: 85px;
+  padding-bottom: 130px;
+  background-color: ${props => props.theme.contact.color.background};
+  color: ${({ theme }) => theme.contact.color.text};
```
Comment (jgwerner) [inline src/templates/pricing.js]: This theme is for contact and not applicable to pricing.

File: src/templates/pricing.js (hunk 1 above)
Comment (reactionic127) [reply src/templates/pricing.js]: Yes you are right. we need to add the sensitive information on pricing query but I just wanted to check how to build is working on this PR

File: src/templates/pricing.js (hunk 2 above)
Comment (reactionic127) [reply src/templates/pricing.js]: I will fix it on next commit

File: src/templates/pricing.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+export default Pricing
+
+export const PricingQuery = graphql`
+  query price($path: String!) {
+    markdownRemark(frontmatter: { path: { eq: $path } }) {
+      frontmatter {
+        title
+        note
+        announcment
```
Comment (jgwerner) [inline src/templates/pricing.js]: Could we fix the spelling and reference for `announcement`?

File: src/pages/pricing.js [hunk 4]
```diff
@@ -31,8 +34,10 @@ export const Price = graphql`
           fileAbsolutePath
           html
           frontmatter {
-            title
             contentType
+            title
+            note
+            announcment
```
Comment (jgwerner) [inline src/pages/pricing.js]: Could we fix the spelling and reference for `announcement`?

File: src/templates/pricing.js (hunk 3 above)
Comment (reactionic127) [reply src/templates/pricing.js]: ok I will fix it

**Comment by jgwerner** [comment]
Merging. Build failing due to upstream/master frontmatter format.

**Pull request merged**